### PR TITLE
Replace PrettyCI by GH-actions

### PR DESCRIPTION
**Pull request by matyo91**

### CODE REVIEW

**Comment by matyo91** [comment]
https://github.com/matyo91/Tornado/pull/2 test pass ✅

Deleted `php7.1` as Swoole could not be installed.
Added `php7.4` to fit the changes of the branche.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+              run: composer install
+
+            - name: 'Tests unit'
+              run: composer tests-unit
+
+            - name: 'Tests examples'
+              run: composer tests-examples
+
+            - name: 'Static analysis'
+              run: composer static-analysis
```
Comment (valentin-claras) [inline .github/workflows/ci.yml:39]: Missing the linter test ;)
```suggestion
              run: composer static-analysis

            - name: 'Code style check'
              run: composer code-style-check
```

**Review by valentin-claras — COMMENTED** [review]
Sorry for the delay. This seems good, but missing php-cs check :)

**Review by b-viguier — APPROVED** [review]
Not sure if swoole extension should be installed in this PR 🤷‍♂️ 
Anyway, this is an important step forward, and sorry for the delay to treat it correctly 😞

**Comment by valentin-claras** [comment]
Merging it now :tada: 
I will update our code style standard in another PR, thanks :bow:

**Pull request merged**